src/main/java/com/github/dockerjava/api/model/Bind.java

```diff
@@ -97,7 +143,15 @@ public PropagationMode getPropagationMode() {
      */
     public static Bind parse(String serialized) {
         try {
-            String[] parts = serialized.split(":");
+
+            String[] parts = null;
+
+            if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
Btw... how that will work when code is create in one jvm (not windows) and executed in other for windows image?
Afair i used this code 1:1 from docker cli code that has no windows specific cases

Reply:
"if (SystemUtils.IS_OS_WINDOWS)" is a runtime evaluation.

If the code is deployed on a Windows host then the control will enter the block.

Reply:
docker daemon may be running on windows, while client on linux. Code shouldn't be dependent. 
What i see is that there is issue with docker cli for mounting c:/ windows paths. How do you do the same with docker cli?

Reply:
BTW the block will only handle the parsing if the token count is 4 or 5, in all other cases it falls back to the default.

The default logic currently expects 2 or 3 tokens.

Reply:
You're right the code won't work if it accesses Docker API remotely.

Reply:
https://github.com/moby/moby/blob/0300fa7f806f978451c09ebbff402ddd34e928d9/volume/windows_parser.go#L300
Seems docker cli has two parsers...

Reply:
Removed the OS check.